test/dynamo/test_modules.py

```diff
@@ -2739,6 +2742,36 @@ def fn(x):
         self.assertEqual(test_functions._variable, 1)
         self.assertEqual(res, 3 * torch.ones(10))
 
+
+    def test_save_and_load1(self):
+        mod = MockModule()
+        opt_mod = torch.compile(mod)
+        inp = torch.randn(10, 10)
+        opt_mod(inp)
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            torch.save(opt_mod, os.path.join(tmpdirname, "model.pt"))
+            loaded_model = torch.load(os.path.join(tmpdirname, "model.pt"))
+            loaded_model(inp)
+            self.assertTrue(same_two_models(loaded_model, mod, [inp]))
+            self.assertTrue(same_two_models(loaded_model, opt_mod, [inp]))
+            assert isinstance(loaded_model, torch._dynamo.OptimizedModule)
+
+    def test_save_and_load2(self):
+        mod = MockModule()
+        opt_mod = torch.compile(mod, backend='inductor')
```

Comment:
Doesn't this test do the same thing as the one above because the default value of backend is inductor?